cli/cmd/testdata/inject_emojivoto_deployment_empty_proxy_version_config.golden.yml

```diff
@@ -0,0 +1,150 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  name: web
+  namespace: emojivoto
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: web-svc
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        linkerd.io/created-by: linkerd/cli dev-undefined
+        linkerd.io/identity-mode: default
+        linkerd.io/proxy-version: test-inject-control-plane-version
```

Comment:
If `proxy.proxy_version` isn't set in the config map, it uses the control plane version as defined by the `global.version` field.